src/view/gui/CMakeLists.txt

```diff
@@ -83,6 +84,10 @@ SET( GUI_SOURCES
   "${CMAKE_CURRENT_SOURCE_DIR}/IPreferencePage.cpp"
   "${CMAKE_CURRENT_SOURCE_DIR}/ApplicationPreferences.h"
   "${CMAKE_CURRENT_SOURCE_DIR}/ApplicationPreferences.cpp"
+  "${CMAKE_CURRENT_SOURCE_DIR}/QZipReader.h"
+  "${CMAKE_CURRENT_SOURCE_DIR}/QZipWriter.h"
+  "${CMAKE_CURRENT_SOURCE_DIR}/StandaloneApps.h"
+  "${CMAKE_CURRENT_SOURCE_DIR}/StandaloneApps.cpp"
```

Comment:
Please delete the 'StandaloneApps' class. We don't need another qsettings handler but just need to extend the existing one.

Reply:
So I just take the implementation over to the ApplicationPreference class?

Reply:
This is already done.